src/containers/AccountConnection.jsx

```diff
@@ -267,13 +275,25 @@ class AccountConnection extends Component {
       return this.updateAccount(account, valuesToSubmit)
     }
 
+    const newAccount = {
+      account_type: konnector.slug,
+      accountType: konnector.slug,
```

Comment:
Why twice the information here?

Reply:
new name with camel case and old name for compatibility ?

Reply:
After a little thinking we should have only one prop at a time, so let's keep account_type for now.